api/swagger-spec/oapi-v1.json

```diff
@@ -18214,6 +18217,10 @@
       "$ref": "v1.RollingDeploymentStrategyParams",
       "description": "input to the Rolling deployment strategy"
      },
+     "after": {
+      "$ref": "v1.AfterDeploymentActions",
```

Comment:
That makes sense, than maybe BeforeAll/AfterAll that seems to have the connotation of being first/last.